packages/autocomplete-core/package.json

```diff
@@ -6,18 +6,32 @@
   "source": "index.tsx",
   "types": "dist/es/index.d.ts",
   "module": "dist/es/index.js",
-  "main": "dist/umd/autocomplete-core.js",
-  "umd:main": "dist/umd/autocomplete-core.js",
-  "unpkg": "dist/umd/autocomplete-core.js",
-  "jsdelivr": "dist/umd/autocomplete-core.js",
+  "main": "dist/umd/index.js",
+  "umd:main": "dist/umd/index.js",
+  "unpkg": "dist/umd/index.js",
+  "jsdelivr": "dist/umd/index.js",
   "homepage": "https://github.com/algolia/autocomplete.js",
   "repository": "algolia/autocomplete.js",
+  "scripts": {
+    "build": "rm -rf ./dist && rollup --config && yarn build:types",
+    "build:types": "tsc -p tsconfig.declaration.json --outDir ./dist/es; echo ''"
+  },
   "author": {
     "name": "Algolia, Inc.",
     "url": "https://www.algolia.com"
   },
   "sideEffects": false,
   "files": [
     "dist/"
-  ]
+  ],
+  "devDependencies": {
```

Comment:
The point of the monorepo is to have all (or most) `devDependencies` top-level so that they're easier to manage.

Reply:
https://github.com/lerna/lerna#common-devdependencies

Ha, I didn't know people do this. I'll update this.

Reply:
https://github.com/francoischalifour/autocomplete.js/pull/26/commits/79f144c83fb18bc4d6755dd6c78d718de4c1e4ae